R/run-model.R

```diff
@@ -94,7 +94,7 @@ hintr_run_model <- function(data, options, output_path = tempfile(),
   progress$complete("Fitting the model")
   progress$start("Generating uncertainty ranges")
   progress$print()
-  fit <- sample_tmb(fit)
+  fit <- sample_tmb(fit, nsample = options$no_of_samples)
```

Comment:
is the full set of these options documented somewhere? is that going to be a `hintr` thing?

Reply:
Not documented. It is specified by naomi in the `inst/extdata/metadata/*_options.json` files. I think if we document that we should probably get it generated from the json automatically or else I can see it getting out of date pretty quick as soon as Jeff changes anything.